tests/Unit/PointwiseFunctions/AnalyticSolutions/Xcts/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ set(LIBRARY "Test_XctsSolutions")
 
 set(LIBRARY_SOURCES
   Test_ConstantDensityStar.cpp
+  Test_HarmonicSchwarzschild.cpp
```

Comment:
(nitpick) Does this need to be placed in alphabetical order (below `Test_Flatness.cpp`)?